lib/instrumental.js

```diff
@@ -233,6 +237,21 @@ exports.init = function instrumental_init(startup_time, config, events) {
       recordCounterRates = config.instrumental.recordCounterRates;
     }
 
+    // do no filtering by default
+    if(typeof(config.instrumental.metricFilters) === 'undefined'){
+      metricFilters = [];
+    } else {
+      metricFilters = config.instrumental.metricFilters;
```

Comment:
Do you think we should do any validation on this? That is, should we verify that this array only contains regular expressions?